node/src/BaseClient.ts

```diff
@@ -3566,11 +3567,53 @@ export class BaseClient {
      * ```
      */
     public zmpop(
-        key: string[],
+        keys: string[],
         modifier: ScoreFilter,
         count?: number,
     ): Promise<[string, [Record<string, number>]] | null> {
-        return this.createWritePromise(createZMPop(key, modifier, count));
+        return this.createWritePromise(createZMPop(keys, modifier, count));
+    }
+
+    /**
+     * Pops a member-score pair from the first non-empty sorted set, with the given `keys` being
+     * checked in the order they are provided. Blocks the connection when there are no members
+     * to pop from any of the given sorted sets. `BZMPOP` is the blocking variant of `ZMPOP`.
+     *
+     * See https://valkey.io/commands/bzmpop/ for more details.
+     *
+     * @remarks
+     *      1. When in cluster mode, all `keys` must map to the same hash slot.
+     *      2. `BZMPOP` is a client blocking command, see https://github.com/valkey-io/valkey-glide/wiki/General-Concepts#blocking-commands
+     *         for more details and best practices.
+     * @param keys - The keys of the sorted sets.
+     * @param modifier - The element pop criteria - either {@link ScoreFilter.MIN} or
+     *     {@link ScoreFilter.MAX} to pop the member with the lowest/highest score accordingly.
+     * @param count - The number of elements to pop.
```

Comment:
Andrew asked me to add more detail for optional parameters on my other PR, not sure if we want to enforce this going forward
```suggestion
     * @param count - (Optional) The number of elements to pop. If not supplied, only one element will be popped.
```